### DISCUSSION TITLE
能否增加倍速播放的快捷键功能？

### DISCUSSION THREAD

**Discussion by MysteryMulberry** [the1812/Bilibili-Evolved#1643] [功能建议 / Ideas]
就像YouTube一样，按"<"键时减一档倍数、按">'时增加一档倍数~谢谢大佬

**Comment by LonelySteve** [comment]
- 启用「快捷键扩展」功能
- 增加一档倍数使用 `Shift + >`，减一档倍数使用 `Shift + <`，也可进行[自定义](https://github.com/the1812/Bilibili-Evolved/blob/preview/src/video/keymap/key-bindings.md)
- 如果觉得自定义键位比较麻烦或者不会弄，也有[一套预设](https://github.com/the1812/Bilibili-Evolved/issues/534#issuecomment-757092836)可以使用（要求 v1.12.1 及以上）：

  ![image](https://user-images.githubusercontent.com/34429322/111032854-42047700-8449-11eb-8c9c-0e5f6760646c.png)

**Comment by Geeyun-JY3** [comment]
你说的这个用 Default 或 YouTube 的预设就是啊……
Default 预设还可以用<kbd>Shift</kbd>+<kbd>↑</kbd> 和 <kbd>Shift</kbd>+<kbd>↓</kbd>

**Comment by gcl-l** [comment]
倍速插件推荐是用 video speed controller 这个插件
自定义快捷键，最高16倍速

**Reply by Geeyun-JY3** [reply]
本脚本也可以自定义快捷键和倍速，最高也是能到 16 倍速：https://github.com/the1812/Bilibili-Evolved/blob/master/doc/features.md#视频
